src/providers/jspm.ts

```diff
@@ -174,15 +176,23 @@ export async function resolveLatestTarget (this: Resolver, target: LatestPackage
 function pkgToLookupUrl (pkg: ExactPackage, edge = false) {
   return `${cdnUrl}${pkg.registry}:${pkg.name}${pkg.version ? '@' + pkg.version : edge ? '@' : ''}`;
 }
-async function lookupRange (this: Resolver, registry: string, name: string, range: string, unstable: boolean, parentUrl?: string): Promise<ExactPackage | null> {
+async function lookupRange (this: Resolver, registry: string, name: string, range: string, semverRange: SemverRange, unstable: boolean, parentUrl?: string): Promise<ExactPackage | null> {
```

Comment:
Let's not add this argument.

Instead, do `const semverRange = new SemverRange(range || '*')` then ensure to use the `unstable` boolean to determine if you want to do unstable or stable inclusion for the range (this is the second argument to the `.has` function - https://www.npmjs.com/package/sver - `range.has(version: Semver|string, unstable = false): bool: Whether the range includes the given version.`)